src/openrct2/object/Object.h

```diff
@@ -42,6 +42,8 @@ enum class ObjectType : uint8_t
     TerrainEdge,
     Station,
     Music,
+    FootpathSurface,
+    FootpathRailings,
```

Comment:
Cause found, this adds two new types, but they are not present in `object_entry_group_counts`, so it reads out of bounds. In release builds that's garbage, possibly very high numbers, hence why it takes so long.

Reply:
Nice, shall I create a PR to fix?